package.json

```diff
@@ -9,6 +9,10 @@
   },
   "author": "Auth0 <support@auth0.com>",
   "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/auth0/express-oauth2-bearer.git"
```

Comment:
Should it just be `https` if it's going to be a link in the npm page?

ref: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository

Reply:
@adamjmcgrath it certainly can be. I’ll change it :)